Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -55,6 +61,8 @@ void SFilledGraphWidget::OnImGuiDraw() const
 
 			ImPlot::EndPlot();
 		}
+
+		ImPlot::PopStyleVar(2);
```

Comment:
I would be tempted to define this '2' number as a `constexpr int32` just before the PushStyleVar calls to hopefully remind anyone Pushing more styles to Pop them too.

Reply:
Oh yes, and with a comment explaining what's going on. Perhaps we can figure out some way of wrapping / scoping these operations so they're less icky and manual? (Which is the API's fault not ours!)